### PR TITLE
Move 'expandableTextbox' to a jenkins-button

### PR DESCRIPTION
**Pull request by mawinter69**
use new button style

affects #8054 where inline js is removed

Before: 
image

After:
image

### Testing done
Functionality is covered by tests
Manual Test:
Create Freestyle job -> Add `Invoke top-level Maven target` build step 

### Proposed changelog entries

- Use new jenkins-button styling for 'expandableTextbox' button 

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/8180"></a>

### CODE REVIEW

File: core/src/main/resources/lib/form/expandableTextbox.jelly
Comment (timja) [inline core/src/main/resources/lib/form/expandableTextbox.jelly:81]: does the `expandable-textbox-button` class do anything? I couldn't see any reference to it

**Review by timja — COMMENTED** [review]
@mawinter69 when saying what testing has been done it would be clearer if you said exactly where.

e.g. for a reviewer it's easier if they don't have to hunt down there this is used.

I tested this in a free style build adding a `Invoke top-level Maven target` step

**Comment by mawinter69** [comment]
Seems the plugin https://plugins.jenkins.io/hp-application-automation-tools-plugin/ has query selectors on the expanding input here:
https://github.com/jenkinsci/hpe-application-automation-tools-plugin/blob/425f167f1f78b8f80adab875150dceebdd8d1044/src/main/webapp/js/specifyParametersUtils.js#L118

**Comment by mawinter69** [comment]
> can you send them a pull request that works for both methods?

sure

**Review by yaroslavafenkin — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
Thanks!

**Review by timja — APPROVED** [review]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by basil** [comment]
Are we confident this will not cause any new PCT failures?

**Comment by NotMyFault** [comment]
> Are we confident this will not cause any new PCT failures?

Yes, https://github.com/jenkinsci/bom/pull/2295

**Comment by basil** [comment]
> @NotMyFault added the `ath-successful` label

Why was this label added when this PR causes 5 consistent and new ATH test failures?

**Pull request merged**
